### ISSUE TITLE
Demo of sidewalkkreator

### ISSUE THREAD

**Issue by Robinlovelace** [udsleeds/openinfra#119]
Heads-up people I've just given this QGIS plugin a spin and seems cool: https://github.com/kauevestena/osm_sidewalkreator/issues/3

Worth exploring further? I'm thinking a vignette with a brief demo could help, could eventually support their docs??

**Comment by hulsiejames** [comment]
This looks very cool - I will download and have a play around and document the journey!

**Comment by Robinlovelace** [comment]
Good stuff, not sure if that's a priority though. Let's have a catch-up tomorrow...

**Comment by hulsiejames** [comment]
Yes agreed, but something to keep in mind! Meeting looks good, acccepted the invite.

**Comment by kauevestena** [comment]
You guys can count on me for any help on the process!
